pynapple/process/signal_processing.py

```diff
@@ -0,0 +1,234 @@
+"""
+Signal processing tools for Pynapple.
+
+"""
+
+from itertools import repeat
+
+import numpy as np
+
+import pynapple as nap
+
+
+def compute_fft(sig, fs):
+    """
```

Comment:
Probably needs more description here. 
Also we need to explicit about the method. There should be a function call `compute_welch_spectrum` or `compute_welch_periodogram` somewhere. It can also be the same function but with different arguments for the methods.